GDCpp/GDCpp/Extensions/Builtin/MouseExtension.cpp

```diff
@@ -48,6 +48,12 @@ MouseExtension::MouseExtension() {
   GetAllConditions()["MouseButtonReleased"]
       .SetFunctionName("MouseButtonReleased")
       .SetIncludeFile("GDCpp/Extensions/Builtin/MouseTools.h");
+   GetAllConditions()["ScrollUp"]  

```

Comment:
You must use the same name: MouseWheelScrollingUp, MouseWheelScrollingDown

Reply:
Oh ok it's the name given in GDCore not in MouseTool 🙈